modules/datasets/samples/pd_inria.cpp

```diff
@@ -53,14 +64,41 @@ int main(int argc, char *argv[])
             cout << "  - ymin = " << y << endl;
             cout << "  - xmax = " << example->bndboxes[j].width + x << endl;
             cout << "  - ymax = " << example->bndboxes[j].height + y << endl;
+
+            Rect r = example->bndboxes[j];
+
+            Rect r0 = r;
+            int vpadding = cvRound(padding * r.height / rheight);
+            r.y = r.y - vpadding;
+            r.height = 2 * vpadding + r.height;
+            r.x = r.x + (r.width / 2);
+            r.width = r.height * rwidth / rheight;
+            r.x = r.x - (r.width / 2);
+
+            if(savebbox)
+            {
+            Mat dst;
+            int top = r0.y - r.y;
+            int bottom = top;
+            int left = r0.x - r.x;
+            int right = left;
+
+            copyMakeBorder(img(r0), dst, top, bottom, left, right, BORDER_REFLECT);
+            resize(dst, dst, Size(rwidth, rheight), 0, 0, INTER_AREA);
+            imwrite(path + format("person_%04d.png", bboxcount++), dst);
+            }
+            else
             rectangle( img, example->bndboxes[j], Scalar( 0, 0, 255 ), 2 );
         }
 
+        if (!savebbox)
+        {
```

Comment:
```
if (!savebbox)
    contrinue;  // skip UI updates
...
```

Reply:
> "skip UI updates" no, on the contrary here we show the image with bounding boxes.

Sure, need to invert condition:

```
if (savebbox)
    continue;  // skip UI updates
```

Or please update indentation of "if body".